src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java

```diff
@@ -265,6 +282,17 @@ public void populateApkSetBuilder(
         .collect(toImmutableList());
   }
 
+  private void logVerboseInfo(ModuleSplit split, ZipPath apkPath) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+    if (verbose) {
+      System.out.printf(
+          "INFO: [%s] '%s' of type '%s' was written to the disk.%n",
+          LocalDateTime.now().format(formatter),
```

Comment:
Use: LocalDateTime.now(ZoneId.systemDefault()) 
to be explicit about which timezone is used.
We have a check that enforces it internally: https://errorprone.info/bugpattern/JavaTimeDefaultTimeZone